src/reducers/recipe-input.reducer.js

```diff
@@ -0,0 +1,49 @@
+
+const initialState = {
+  name: '',
+  ingredients: [],
+  description: [],
+};
+
+export default (state = initialState, action) => {
+  let newDescription = [...state.description];
```

Comment:
Agree with @nickschoey which approach to follow in the reducers.
Personally, I prefer to crate the new objects when I need them, often directly in the `return` statement.